types/module/module.go

```diff
@@ -216,6 +218,26 @@ type HasServices interface {
 	RegisterServices(Configurator)
 }
 
+// HasServices is the extension interface that modules should implement to register
```

Comment:
Why cannot it stay in core (as it is module related), but under a build flag that does not get used when we build for tinygo?

I don't think we should add anything to the SDK as it just moves the problem.

Reply:
This imports grpc we can't have core importing grpc

Reply:
Right, but if you build it with a specific build flag, it won't import grpc right?

Reply:
The ux around adding tags for this sort of thing is wonky. You'd need to add flags to core and the servers. I think it's cleaner to get core working with no tags.

Reply:
removed that move so we can get this merged. But grpc has to go from this dependency, there isnt really a way around it